someip2val/cert/getCerts.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
```

Comment:
No reason to include this file - better to mention https://github.com/eclipse/kuksa.val/blob/master/kuksa_certificates/genCerts.sh in a README.md?